ros_package_template/src/template_ros_pub.cpp

```diff
@@ -0,0 +1,34 @@
+#include <ros_package_template/template_ros_pub.hpp>
+
+#include <pcl_conversions/pcl_conversions.h>
+
+namespace ariitk::ros_package_template {
+
+void TemplateROSPub::init(ros::NodeHandle& nh) {
```

Comment:
everything under namespace should be indented right

Reply:
Yes you are right

Reply:
no. please support argument with a reference

Reply:
@gajena we discussed this today. stuff inside namespaces isn’t indented

Reply:
then the class inside the namespace in ros_package_template/include/ros_package_template/template_ros.hpp should not be indented as well, right?

Reply:
> @gajena we discussed this today. stuff inside namespaces isn’t indented

Oh damn, I read it wrong. Yeah, it should not be. And correct your all files some has the indentation